DisCatSharp.Docs/articles/misc/hosting.md

```diff
@@ -22,6 +22,65 @@ Any modern hardware will work just fine, new or used.
 
 Depending on how complex your bot is, you may even consider purchasing a Raspberry Pi ($35).
 
+# Termux
+
+For those who don't have some PC or other hardware lying around alternatively you could use your phone. Using [Termux](https://termux.dev/en/) and a package called [proot-distro](https://github.com/termux/proot-distro), which we use to install a Debian vm and setup DotNET to run the bot. To those interested the steps are given below:
+
+## Requirements
+
+- An Android 5+
+- Termux
+- An internet connection
+
+## Setup
+
+- Initialize Termux
+```sh
+pkg update && pkg upgrade -y
+```
+*Note, it might ask you for input, just click enter and let the default option execute*
+
+- Install proot-distro package
+```sh
+pkg install proot-distro -y
+```
+
+- Install a Ubuntu vm
```

Comment:
I feel vm should be typed out fully, for clarity.
Also as discussed on Discord, please look into using Debian instead because it's easier to run.

Reply:
Got it, looking into Debian currently